src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/components/structure/page/customheaderlibs.html

```diff
@@ -18,3 +18,20 @@
 
 <!--/* Include Context Hub */-->
 <sly data-sly-resource="${symbol_dollar}{'contexthub' @ resourceType='granite/contexthub/components/contexthub'}"/>
+<!--/* Data layer JavaScript object: */-->
+<script  data-sly-use.data="/apps/${appsFolderName}/components/structure/page/digitaldata.js"
```

Comment:
-1 for using JS-use API. All information is already available in the context.

Reply:
Resolved.